contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
+    default: "/opt/stackstorm/packs/examples/actions/workflows/print_text.sl"
+    type: string
+  inputs:
```

Comment:
I agree with this approach.
If I understand right that is how Mistral did as well, what we wanted to do first is to enable to run any cloudslang flow, hence the `inputs` and `path` keywords and later on create specific action packs for cloudslang.
That being said, I agree that the print example could and should use `text` as input and not `inputs`.
anyway to retrieve the inputs dynamically? if it is possible, we will be able to fetch the inputs from cloudslang for a given path instead of wrapping all of our cloudslang content in actions.